tools/licenses/lib/main.dart

```diff
@@ -1044,7 +1044,6 @@ class _RepositoryDirectory extends _RepositoryEntry implements LicenseSource {
     '/third_party/libcxx/LICENSE.TXT': _RepositoryCxxStlDualLicenseFile.new,
     '/third_party/libcxxabi/LICENSE.TXT': _RepositoryCxxStlDualLicenseFile.new,
     '/third_party/libpng/LICENSE': _RepositoryLibPngLicenseFile.new,
-    '/third_party/root_certificates/LICENSE': _RepositoryMpl2File.new,
```

Comment:
Wondering if this will need to be updated to the new path rather than removed, but if the license script is still happy this lgtm.

Reply:
Yeah, the script seems to be happy without this.